OKTA-353663 - Update Policy API Default Rules (#4214)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/reference/api/policy/index.md

### Default Rules

>  - Only the default Policy contains a default Rule. For the Okta Classic Engine, default rules can’t be deleted or edited. For the Okta Identity Engine, default rules can’t be deleted, but can be edited except in the case of the default rule on the Authenticator Enrollment policy and the Identity Provider Routing.

Reviewer: ```suggestion
 - Only the default Policy contains a default Rule. In Okta Classic Engine, you can't delete or edit default rules. In Okta Identity Engine, you can't delete default rules, but can edit them except in the case of the default rule on the Authenticator Enrollment policy and the Identity Provider Routing.
```
